### ISSUE TITLE
[Airbrake] bad URI(is not URI?): /v1/cameras/hikvision ds-2cd8133f-e.json

### ISSUE THREAD

**Issue by marcoherbst** [evercam/evercam-dashboard#1211]
**Airbrake error:** [#3204](https://evercam.airbrake.io/projects/98599/groups/1687605070647163204)
**Airbrake project:** Evercam Dashboard

**Error type:** `URI::InvalidURIError`
**Error message:** `` bad URI(is not URI?): /v1/cameras/hikvision ds-2cd8133f-e.json ``
**Where:** `cameras#single`
**Occurred at:** May 17, 2016 10:51:23 UTC
**First seen at:** May 17, 2016 10:51:23 UTC
**Occurrences:** 1 (0 since last deploy on `<no information>`)

**URL:** [https://dash.evercam.io/v1/cameras/hikvision%20ds-2cd8133f-e.json](https://dash.evercam.io/v1/cameras/hikvision%20ds-2cd8133f-e.json)
**File:** `/app/vendor/ruby-2.3.0/lib/ruby/2.3.0/uri/generic.rb`

**Backtrace:**
```
/app/vendor/ruby-2.3.0/lib/ruby/2.3.0/uri/generic.rb:1102:in rescue in merge
/app/vendor/ruby-2.3.0/lib/ruby/2.3.0/uri/generic.rb:1099:in merge
[GEM_ROOT]/gems/faraday-0.9.2/lib/faraday/connection.rb:406:in build_exclusive_url
[GEM_ROOT]/gems/faraday-0.9.2/lib/faraday/rack_builder.rb:192:in build_env
...
Caused by URI::InvalidURIError: bad URI(is not URI?): /v1/cameras/hikvision ds-2cd8133f-e.json
```

**Comment by muneeb2542** [comment]
This is a bad url error happened only because of intentionally writing the above url in the browser.I will configure airbrake to ignore such errors.
So it is irrelevant now.Closing this

**Comment by marcoherbst** [comment]
100% sure? ... it is not a " " character in the model ID ?

if so, that's fine, close it.

m

On 20 May 2016 at 08:46, Muneeb Rana <notifications@github.com> wrote:

> Closed #1211 <https://github.com/evercam/evercam-dashboard/issues/1211>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/evercam/evercam-dashboard/issues/1211#event-666804764>
>



-- 
Marco Herbst
*Evercam.io <http://www.evercam.io/>* - Cameras Unleashed
Mobile: +353 86 196 0451
Skype: marco.a.herbst

**Comment by muneeb2542** [comment]
Yes, I am 100% sure.This only happened because of manually writing the
above URL in the browser.

Plus the vendor and model requests we use don't have model id or vendor id
in the URL.

On 20 May 2016 at 12:57, Marco Herbst <notifications@github.com> wrote:

> 100% sure? ... it is not a " " character in the model ID ?
>
> if so, that's fine, close it.
>
> m
>
> On 20 May 2016 at 08:46, Muneeb Rana <notifications@github.com> wrote:
>
> > Closed #1211 <https://github.com/evercam/evercam-dashboard/issues/1211>.
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/evercam/evercam-dashboard/issues/1211#event-666804764>
> >
>
>
>
> --
> Marco Herbst
> *Evercam.io <http://www.evercam.io/>* - Cameras Unleashed
> Mobile: +353 86 196 0451
> Skype: marco.a.herbst
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/evercam/evercam-dashboard/issues/1211#issuecomment-220540022>
>



-- 


Muneeb-ur-Rehman
Junior Software Developer
Evercam.io
